.circleci/config.yml

```diff
@@ -9,18 +12,34 @@ executors:
       - image: circleci/python:<<parameters.python_version>>
 
 commands:
+##
+  setup_python_env:
+    description: Setup Python environment
+    steps:
+      - run:
+          name: Install Python
+          command: python --version
+      - run:
+          name: Install Python dependencies
+          command: |
+            pip install git+https://github.com/kedro-org/kedro@main
+            cd package && pip install -r requirements.txt -U
```

Comment:
Where is this `package` folder coming from?

Reply:
Nice catch - looks like we're installing kedro and it's requirements, and then installing them again under the kedro-starters requirements; I'll remove the redundant step.